agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4553,6 +4568,38 @@ bool slave_thread::handle_ap_metrics_query(Socket *sd, ieee1905_1::CmduMessageRx
         LOG(DEBUG) << "Received AP_METRICS_QUERY_MESSAGE, mid=" << std::hex << int(mid)
                    << "  bssid " << std::get<1>(bssid);
     }
+    LOG(DEBUG) << "Forwarding AP_METRICS_QUERY_MESSAGE to monitor_socket, mid=" << std::hex
+               << int(mid);
+    uint16_t length = message_com::get_uds_header(cmdu_rx)->length;
+    cmdu_rx.swap(); // swap back before forwarding
+    if (!message_com::forward_cmdu_to_uds(monitor_socket, cmdu_rx, length)) {
+        LOG(ERROR) << "Failed sending AP_METRICS_QUERY_MESSAGE message to monitor_socket";
+        return false;
+    }
+    return true;
+}
+
+bool slave_thread::handle_ap_metrics_response(Socket &sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received AP_METRICS_RESPONSE_MESSAGE, mid=" << std::hex << int(mid);
+
+    auto ap_metrics_response_tlv = cmdu_rx.getClass<wfa_map::tlvApMetric>();
+    if (!ap_metrics_response_tlv) {
+        LOG(ERROR) << "Failed cmdu_rx.getClass<wfa_map::tlvApMetric>()" << std::hex << int(mid);
+        return false;
+    }
```

Comment:
Since we don't do anything with this, it's not needed to check it. Just forward it.